server.js

```diff
@@ -0,0 +1,6 @@
+const http = require('http');
+const app = require('./lib/app');
+
+http.createServer(app)
+  .listen(7890);
```

Comment:
```suggestion
  .listen(PORT, () => console.log(`Server running on ${PORT}`));
```
Can help to have a visual that your server is running (and on what port)